frontend/src/views/FuelCodes/AddFuelCode/AddFuelCode.jsx

```diff
@@ -1,3 +1,57 @@
+import BCBox from '@/components/BCBox'
+import { columnDefs, defaultColDef } from './_schema'
+import { useRef, useState, useCallback } from 'react'
+import BCDataGridEditor from '@/components/BCDataGrid/BCDataGridEditor'
+import { Typography } from '@mui/material'
+
 export const AddFuelCode = () => {
-  return <div>AddFuelCode</div>
+  const [rowData, setRowData] = useState(null)
+  const [gridApi, setGridApi] = useState(null)
+  const [columnApi, setColumnApi] = useState(null)
+  const gridRef = useRef(null)
+
+  function onGridReady(params) {
+    setGridApi(params.api)
+    setColumnApi(params.columnApi)
+
+    fetch('http://localhost:4000/fuelCodes')
```

Comment:
this url doesn't look correct? and add some error handling here

Reply:
This too shall be modified as part of [#432](https://github.com/bcgov/lcfs/issues/432).